CRM/Dedupe/Merger.php

```diff
@@ -586,36 +586,23 @@ public static function retrieveFields($main, $other) {
    *                              A 'safe' value skips the merge if there are any un-resolved conflicts.
    *                              Does a force merge otherwise.
```

Comment:
Could we add a comment here that makes it explicit that the two strings accepted for the mode are 'safe' and 'aggressive' ?